tests/utils_meshes_test.py

```diff
@@ -0,0 +1,98 @@
+# ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+#  Copyright (c) 2024 Mira Geoscience Ltd.
+#  All rights reserved.
+#
+#  This file is part of simpeg-drivers.
+#
+#  The software and information contained herein are proprietary to, and
+#  comprise valuable trade secrets of, Mira Geoscience, which
+#  intend to preserve as trade secrets such software and information.
+#  This software is furnished pursuant to a written license agreement and
+#  may be used, copied, transmitted, and stored only in accordance with
+#  the terms of such license and with the inclusion of the above copyright
+#  notice.  This software and information or any other copies thereof may
+#  not be provided or otherwise made available to any other person.
+#
+# ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+import numpy as np
+from geoh5py import Workspace
+from geoh5py.objects import Points
+from perlin_noise import PerlinNoise
```

Comment:
Numpy random won't give anything resembling topography, so it's not a good alternative.  I can probably just provide a flat topo now though since I abandoned the gradient based approach.

Reply:
Using scipy gaussian_filter with a randn to get a random topo now.